### ISSUE TITLE
Add links from examples to online code editor

### ISSUE THREAD

**Issue by ivmartel** [ankitskvmdam/clean-jsdoc-theme#41]
`@example` code could provide a link to online editors using their post API, see for example [stackblitz post-api](https://developer.stackblitz.com/docs/platform/post-api) or [codepen prefill](https://blog.codepen.io/documentation/prefill/).

**Comment by ankitskvmdam** [comment]
Do you have implementation ideas?

**Comment by ivmartel** [comment]
In the examples I have seen, you have to create a form with hidden fields containing the 'code' data. For example in codepen:

```html
<form action="https://codepen.io/pen/define" method="POST" target="_blank">
  <input type="hidden" name="data" value='{"title": "New Pen!", "html": "<div>Hello, World!</div>"}'>
  <input type="submit" value="Create New Pen with Prefilled Data">
</form>
```

Couldn't that be integrated in the [example.tmpl](../blob/master/tmpl/example.tmpl)?

**Comment by ankitskvmdam** [comment]
Yes, it can be implemented.

**Comment by ivmartel** [comment]
Hi, is this feature available in recent versions?

**Comment by ankitskvmdam** [comment]
No, this feature is not implemented.

**Comment by ivmartel** [comment]
Ok, and do you have plans for it?

**Comment by ankitskvmdam** [comment]
I will try to work on it this weekend or would you like to work on this?

**Comment by ankitskvmdam** [comment]
@ivmartel I have added the option to open example code in codepen. But this feature is not enabled by default. [Visit here](https://github.com/ankitskvmdam/clean-jsdoc-theme#codepen-) for more information on how to enable this feature. Also, this feature will be available in v3.2.5

**Comment by ivmartel** [comment]
Looking good! thanks! 

I just have troubles passing HTML with quotes (something like `<div id="a"></div>`) in the codepen `html` option, tried single quote or escaping but with no success...

**Comment by ankitskvmdam** [comment]
Currently you have to do something like this: `<div id=&apos;a&apos;></div>`

**Comment by ankitskvmdam** [comment]
But in a future release, you don't have to do it like this. You can do like: `<div id="a"></div>`. You can expect this fix in v3.2.7
